google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsConstants.java

```diff
@@ -0,0 +1,266 @@
+
+  private static final Aggregation AGGREGATION_WITH_MILLIS_HISTOGRAM =
+      Aggregation.explicitBucketHistogram(
+          ImmutableList.of(
+              0.0, 0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 8.0, 10.0, 13.0,
+              16.0, 20.0, 25.0, 30.0, 40.0, 50.0, 65.0, 80.0, 100.0, 130.0, 160.0, 200.0, 250.0,
+              300.0, 400.0, 500.0, 650.0, 800.0, 1000.0, 2000.0, 5000.0, 10000.0, 20000.0, 50000.0,
+              100000.0, 200000.0, 400000.0, 800000.0, 1600000.0, 3200000.0)); // max is 53.3 minutes
+
+  static final String SCOPE = "bigtable.googleapis.com";
+
+  static final InstrumentSelector OPERATION_LATENCIES_SELECTOR =
+      InstrumentSelector.builder()
+          .setName(OPERATION_LATENCIES_NAME)
+          .setMeterName(SCOPE)
+          .setType(InstrumentType.HISTOGRAM)
+          .setUnit("ms")
+          .build();
+  static final InstrumentSelector ATTEMPT_LATENCIES_SELECTOR =
+      InstrumentSelector.builder()
+          .setName(ATTEMPT_LATENCIES_NAME)
+          .setMeterName(SCOPE)
+          .setType(InstrumentType.HISTOGRAM)
+          .setUnit("ms")
+          .build();
+
+  static final InstrumentSelector RETRY_COUNT_SELECTOR =
+      InstrumentSelector.builder()
+          .setName(RETRY_COUNT_NAME)
+          .setMeterName(SCOPE)
+          .setType(InstrumentType.COUNTER)
+          .setUnit("1")
+          .build();
+
+  static final InstrumentSelector SERVER_LATENCIES_SELECTOR =
+      InstrumentSelector.builder()
+          .setName(SERVER_LATENCIES_NAME)
+          .setMeterName(SCOPE)
+          .setType(InstrumentType.HISTOGRAM)
+          .setUnit("ms")
+          .build();
+  static final InstrumentSelector FIRST_RESPONSE_LATENCIES_SELECTOR =
+      InstrumentSelector.builder()
+          .setName(FIRST_RESPONSE_LATENCIES_NAME)
+          .setMeterName(SCOPE)
+          .setType(InstrumentType.HISTOGRAM)
+          .setUnit("ms")
+          .build();
+  static final InstrumentSelector CLIENT_BLOCKING_LATENCIES_SELECTOR =
+      InstrumentSelector.builder()
+          .setName(CLIENT_BLOCKING_LATENCIES_NAME)
+          .setMeterName(SCOPE)
+          .setType(InstrumentType.HISTOGRAM)
+          .setUnit("ms")
+          .build();
+  static final InstrumentSelector APPLICATION_BLOCKING_LATENCIES_SELECTOR =
+      InstrumentSelector.builder()
+          .setName(APPLICATION_BLOCKING_LATENCIES_NAME)
+          .setMeterName(SCOPE)
+          .setType(InstrumentType.HISTOGRAM)
+          .setUnit("ms")
+          .build();
+  static final InstrumentSelector CONNECTIVITY_ERROR_COUNT_SELECTOR =
+      InstrumentSelector.builder()
+          .setName(CONNECTIVITY_ERROR_COUNT_NAME)
+          .setMeterName(SCOPE)
+          .setType(InstrumentType.COUNTER)
+          .setUnit("1")
+          .build();
+
+  public static final View OPERATION_LATENCIES_VIEW =
+      View.builder()
+          .setName("bigtable.googleapis.com/internal/client/operation_latencies")
```

Comment:
There is quite a bit of duplication in this file:
- `static final String SCOPE = "bigtable.googleapis.com";`
- View.builder()
          .setName("bigtable.googleapis.com/internal/client/operation_latencies")
-   InstrumentSelector.builder()
          .setName(OPERATION_LATENCIES_NAME)
          .setMeterName(SCOPE)


It would be a lot nicer if you can create a helper that takes the core parameters and applies everything:
```
void defineView(String id, Aggregation agg, Type type, String unit, List<Attribute> extraAttributes( {
   InstrumentSelector selector =  InstrumentSelector.builder()
          .setName(id)
          .setMeterName(SCOPE)
          .setType(type)
          .setUnit(unit)
          .build();
  attrs = ImmutableList.builder() 
     .addAll(COMMON_ATTRIBUTES)
     .addAll(extraAttributes)
     .build()
   View view =
      View.builder()
          .setName(METER_NAME + "/internal/client/" + id)
          .setAggregation(agg)
          .setAttributeFilter(attrs);

   map.put(selector, view)
}

```